test/plugin/base_test.rb

```diff
@@ -526,16 +631,16 @@ def test_one_log_ec2
     verify_log_entries(1, EC2_PARAMS)
   end
 
-  def test_struct_payload_log
+  def test_json_payload_log
```

Comment:
I wonder if we should just change s/struct_payload/structured_payload/ instead of switching to "json_payload"... WDYT?